internal/scaffold/ansible/travis.go

```diff
@@ -32,7 +32,8 @@ func (t *Travis) GetInput() (input.Input, error) {
 	return t.Input, nil
 }
 
-const travisAnsibleTmpl = `sudo: required
+const travisAnsibleTmpl = `---
+sudo: required
```

Comment:
I don't think the `sudo: required` flag is required anymore on Travis CI to get the Molecule-based tests to run correctly (just confirmed in a test on my Tower operator role, see: https://travis-ci.com/geerlingguy/tower-operator/builds/145256598, commit: https://github.com/geerlingguy/tower-operator/commit/b7312390ab8f94bde29faf5ad57642c20a082a4c).